oval_graph/oval_tree/evaluate.py

```diff
@@ -144,26 +219,58 @@ def one_is_unknown(result):
 
 
 def one_is_error(result):
+    """Evaluates if the result match unknown for one operator.
+
+    Args:
+            result (dict): Dictionary representing frequencies of values
+
+    Returns:
+            bool.
+    """
     return (smaller_than_two(result, 'true_cnt')
             and eq_or_greater_zero(result, 'false_cnt')
             and greater_zero(result, 'error_cnt')
             and eq_or_greater_zero_unknown_noteval_notappl(result))
 
 
 def one_is_false(result):
+    """Evaluates if the result match false for one operator.
+
+    Args:
+            result (dict): Dictionary representing frequencies of values
+
+    Returns:
+            bool.
+    """
     return (eq_zero(result, 'true_cnt')
             and eq_or_greater_zero(result, 'false_cnt')
             and error_unknown_noteval_eq_zero(result)
             and result['notappl_cnt'] >= 0)
 
 
 def one_is_false1(result):
```

Comment:
What's the difference between this and the `one_is_false` one?

Reply:
Operator ONE has two cases of false state. I merge it into one function.